packages/ssr/src/analytics-events.ts

```diff
@@ -68,6 +69,17 @@ const sendAnalyticsEvents = async (
       sendingEvents.push(...listMetaEvents);
     }
   }
+  if (eventsByType.ttk) {
+    const user = {
+      ip: payload.id,
```

Comment:
```suggestion
      ip: payload.ip,
```

Aliás agora que vi que tá errado no do Meta também...